client/ayon_nuke/api/lib.py

```diff
@@ -810,89 +812,24 @@ def check_inventory_versions():
     and check if the node is having actual version. If not then it will color
     it to red.
     """
```

Comment:
Admittedly this docstring is weird, no? Should we update it with this PR?

```
    """Update loaded container nodes' colors based on version state.

    This will group containers by their version to outdated, not found,
    invalid or latest and colorize the nodes based on the category.
    """
```

Reply:
I agree, updating it now